tests/typed-document-node.spec.ts

```diff
@@ -16,14 +16,18 @@ function findAncestor(node: ts.Node, visitor: (n: ts.Node) => boolean) {
   return found;
 }
 
-describe('TypedDocumentNode', () => {
-  it('Should not output imports when there are no operations at all', async () => {
-    const result = (await plugin(null as any, [], {})) as Types.ComplexPluginOutput;
-    expect(result.content).toBe('');
+describe("TypedDocumentNode", () => {
+  it("Should not output imports when there are no operations at all", async () => {
+    const result = (await plugin(
+      null as any,
+      [],
+      {}
+    )) as Types.ComplexPluginOutput;
+    expect(result.content).toBe("");
     expect(result.prepend.length).toBe(0);
   });
 
-  it('Should not output object literal', async () => {
+  it("Should not output object literal", async () => {
```

Comment:
are any of these changes in `tests/typed-document-node.spec.ts` necessary?

Reply:
I can roll them back, running prettier would avoid this.